routers/web/web.go

```diff
@@ -842,19 +848,23 @@ func RegisterRoutes(m *web.Route) {
 					m.Post("/initialize", web.Bind(forms.InitializeLabelsForm{}), org.InitializeLabels)
 				})
 
-				m.Group("/runners", func() {
-					m.Get("", org.Runners)
-					m.Combo("/{runnerid}").Get(org.RunnersEdit).
-						Post(web.Bind(forms.EditRunnerForm{}), org.RunnersEditPost)
-					m.Post("/{runnerid}/delete", org.RunnerDeletePost)
-					m.Get("/reset_registration_token", org.ResetRunnerRegistrationToken)
-				}, actions.MustEnableActions)
+				m.Group("/actions", func() {
+					m.Get("", func(ctx *context.Context) {
```

Comment:
I would propose to have a func with name for it, instead of writing too much code in this "route setup" function.

Reply:
changed in https://github.com/go-gitea/gitea/commit/90fea6c33591f808e5a9811742eba546f3fa99f2